proof/refine/ARM/Finalise_R.thy

```diff
@@ -2326,6 +2326,342 @@ lemma replyClear_invs'[wp]:
   apply wpsimp
   sorry
 
+lemma isHeadSome:
+ "isHead (Some h) \<longleftrightarrow> (\<exists>p. h = Head p)"
+  by (clarsimp simp: isHead_def split: reply_next.split)
```

Comment:
This doesn’t seem to be used, and it is pretty much the same as existing `isHead_to_head`. So, probably we should remove this.

Reply:
Removed